hmmlearn/utils.py

```diff
@@ -60,3 +60,9 @@ def iter_from_X_lengths(X, lengths):
 
         for i in range(len(lengths)):
             yield start[i], end[i]
+
+def prob_log(prob):
```

Comment:
As I checked the example from #33, I didn't see the patch break non-ergodic transition 
matrices model.
================================
`transmat and startprob BEFORE:
[[ 0.5  0.5  0. ]
 [ 0.   0.5  0.5]
 [ 0.   0.   1. ]]
[ 1.  0.  0.]


transmat and startprob AFTER:
[[ 0.50371  0.49629  0.     ]
 [ 0.       0.00117  0.99883]
 [ 0.       0.       1.     ]]
[ 1.  0.  0.]`